src/cohort.js

```diff
@@ -0,0 +1,114 @@
+class Cohort {
+  constructor() {
+    this.cohortObject = {
+      cohortName: '',
+      studentList: []
+    }
+    this.cohortList = []
+    this.allStudents = []
+  }
+
+  findCohort(cohortName) {
+    const cohort = this.cohortList.find(
+      (cohort) => cohort.cohortName === cohortName
+    )
+    if (!cohort) {
+      return 'cohort does not exist'
```

Comment:
here instead you could have it throw an error by replacing line 16 with "throw new Error('cohort does not exist')

Reply:
Yeah I've seen that in a few other peoples code - I need to look into the throw error syntax